src/program/snabbvmx/lwaftr/lwaftr.lua

```diff
@@ -140,19 +141,20 @@ function run(args)
       lwaftr_id.value = id
    end
 
-   local vlan = effective_vlan(conf, lwconf)
-
+   local vlan = false
```

Comment:
Its default is false if there is no config.  This default is overridden by the block that follows.